bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/battery.xml

```diff
@@ -17,6 +17,7 @@
 		<description>Capacity of this battery</description>
 		<category>Energy</category>
 		<state pattern="%.1f %unit%" readOnly="true"/>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
```

Comment:
readOnly channel should not be annotated with autoUpdatePolicy, as they can typically not be set.

Reply:
For the writable channels this will make sure that there is no jumping between the states if the API call wasn't successful.

Reply:
Got it, you are right. Will remove it for all channels then.